### ISSUE TITLE
Remove isOsiApproved for MIT-0

### ISSUE THREAD

**Issue by swinslow** [spdx/license-list-XML#957]
The [MIT-0 submission request](https://github.com/spdx/license-list-XML/issues/619) noted that MIT-0 was not OSI approved. 

However, [the template file](https://github.com/spdx/license-list-XML/blob/master/src/MIT-0.xml) has `isOsiApproved="true"`, which appears to have been included inadvertently (perhaps through copying from the MIT template, but I'm just guessing).

Per [1](https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2019-December/004576.html) and [2](https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2019-December/004577.html), it appears that MIT-0 has not in fact been approved by OSI. This should be confirmed and, if correct, the `isOsiApproved` tag should be removed from MIT-0 for the 3.8 release.

**Comment by smasher164** [comment]
I grepped through all of the mailing list archives, and found no associated records.
```
$ ls
2007-December.txt  2009-December.txt  2010-July.txt      2011-September.txt 2013-June.txt      2014-September.txt 2016-June.txt      2017-October.txt   2019-January.txt
2008-April.txt     2009-February.txt  2010-June.txt      2012-April.txt     2013-March.txt     2015-August.txt    2016-March.txt     2017-September.txt 2019-July.txt
2008-August.txt    2009-January.txt   2010-May.txt       2012-August.txt    2013-May.txt       2015-December.txt  2016-May.txt       2018-April.txt     2019-June.txt
2008-December.txt  2009-July.txt      2010-November.txt  2012-February.txt  2013-November.txt  2015-February.txt  2016-November.txt  2018-December.txt  2019-March.txt
2008-February.txt  2009-June.txt      2010-October.txt   2012-January.txt   2013-October.txt   2015-January.txt   2016-October.txt   2018-February.txt  2019-May.txt
2008-January.txt   2009-March.txt     2010-September.txt 2012-July.txt      2013-September.txt 2015-July.txt      2017-April.txt     2018-June.txt      2019-November.txt
2008-July.txt      2009-May.txt       2011-April.txt     2012-June.txt      2014-April.txt     2015-June.txt      2017-August.txt    2018-March.txt     2019-October.txt
2008-June.txt      2009-November.txt  2011-August.txt    2012-March.txt     2014-August.txt    2015-November.txt  2017-December.txt  2018-May.txt       2019-September.txt
2008-March.txt     2009-October.txt   2011-December.txt  2012-May.txt       2014-December.txt  2015-October.txt   2017-February.txt  2018-November.txt  main.go
2008-May.txt       2009-September.txt 2011-July.txt      2012-September.txt 2014-July.txt      2015-September.txt 2017-January.txt   2018-October.txt
2008-November.txt  2010-April.txt     2011-June.txt      2013-April.txt     2014-June.txt      2016-April.txt     2017-July.txt      2018-September.txt
2008-October.txt   2010-August.txt    2011-March.txt     2013-December.txt  2014-March.txt     2016-August.txt    2017-June.txt      2019-April.txt
2008-September.txt 2010-December.txt  2011-May.txt       2013-February.txt  2014-May.txt       2016-December.txt  2017-March.txt     2019-August.txt
2009-April.txt     2010-February.txt  2011-November.txt  2013-January.txt   2014-November.txt  2016-February.txt  2017-May.txt       2019-December.txt
2009-August.txt    2010-January.txt   2011-October.txt   2013-July.txt      2014-October.txt   2016-January.txt   2017-November.txt  2019-February.txt
```
```
$ grep -i "mit-0\|mit-zero\|no attribution" *
```

Here is the script I ran to download all of the mailing lists.
```
package main

import (
	"compress/gzip"
	"fmt"
	"io"
	"log"
	"net/http"
	"os"
	"strconv"
	"time"
)

func main() {
	const prefix = "https://lists.opensource.org/pipermail/license-review_lists.opensource.org/"
	for y := 2007; y <= 2019; y++ {
		year := strconv.Itoa(y)
		for m := time.January; m <= time.December; m++ {
			name := fmt.Sprintf("%s-%s.txt.gz", year, m)
			u := prefix + name
			resp, err := http.Get(u)
			if err != nil {
				log.Println(err)
				continue
			}
			defer resp.Body.Close()
			if resp.StatusCode == 404 {
				log.Printf("404 on %s", name)
				continue
			}
			out, err := os.Create(name[:len(name)-3])
			if err != nil {
				log.Println(err)
				continue
			}
			r, err := gzip.NewReader(resp.Body)
			if err != nil {
				log.Println(err)
				continue
			}
			_, err = io.Copy(out, r)
			if err != nil {
				log.Println(err)
			}
		}
	}
}
```

**Comment by swinslow** [comment]
Thanks @smasher164 for catching this and for thoroughly researching it  :)

I've merged #958 -- it will be updated on [the license list page](https://spdx.org/licenses) upon the 3.8 release of the license list, which has been delayed but will be upcoming.
